server/services/nps/sendNpsResults.ts

```diff
@@ -27,7 +27,7 @@ export const sendNpsResults = Meteor.bindEnvironment(function sendNpsResults(nps
 			data,
 		});
 	} catch (e) {
-		SystemLogger.error(e);
+		SystemLogger.error(e as Error);
```

Comment:
depends on https://github.com/RocketChat/Rocket.Chat/pull/24726

```suggestion
		SystemLogger.error(e);
```